study.md

```diff
@@ -152,7 +152,12 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+Resources: expose simple directory structre URIs
```

Comment:
These aren't HTTP Verbs. Those are principles of REST.

That resource lists the verbs right under what you copied.